examples/kubernetes/dynamic_provisioning/README.md

```diff
@@ -28,6 +28,7 @@ parameters:
 * dataCompressionType (Optional) - FSx for Lustre supports data compression via LZ4 algorithm. Compression is disabled when the value is set to NONE. The default value is NONE 
 * weeklyMaintenanceStartTime (Optional) - The preferred start time to perform weekly maintenance, formatted d:HH:MM in the UTC time zone, where d is the weekday number, from 1 through 7, beginning with Monday and ending with Sunday. The default value is "7:09:00" (Sunday 09:00 UTC)
 * fileSystemTypeVersion (Optional) - Sets the Lustre version of the Amazon FSx for Lustre file system to be created. Valid values are 2.10 and 2.12. The default value is "2.10"
+* awsTags (Optional) - Tags that will be set on the FSx resource created in AWS, in the form of a comma separated list with each tag delimited by a colon (example - "Tag1:Value1,Tag2:Value2") . Default is a single tag with CSIVolumeName as the key and the generated volume name as it's value.
```

Comment:
I think it would be beneficial to have this similar to the EBS CSI Driver. See https://github.com/kubernetes-sigs/aws-ebs-csi-driver/blob/master/cmd/options/controller_options.go#L43
i.e flag is `--extra-tags` and it is on the form `Tag1=value1,Tag2=value2`

Not huuugely important since e.g load balancer controller calls the flag `default-tags`. See https://github.com/kubernetes-sigs/aws-load-balancer-controller/blob/6c706b023937e20f6dd7b2ea5e0761d55017f772/pkg/config/controller_config.go#L104

Reply:
Renamed `awsTags` to `extraTags`